app/Http/Controllers/FrontController.php

```diff
@@ -398,6 +398,10 @@ public function storeVisit(Request $request) {
 
     public function newFeedback($controllerSelected=null) {
         $feedbackOptions = User::where('status', 1)->orderBy('lname', 'ASC')->get()->pluck('backwards_name', 'id');
+        $feedbackCIDs = User::where('status', 1)->orderBy('id', 'ASC')->get()->pluck('id');
```

Comment:
This is fine if this is how we have to do it, but I'd like for there to be a way to do this without grabbing all of them, and then iterating through them. Like is there a version of pluck() that'll allow us to append a c to the beginning of the CID and also have the plain CID?

Reply:
@iccowan Agreed - in fact, that is what I tried to do before I wrote it this way. I couldn't find a simpler way of just reusing the first query. Unfortunately when using Laravel's abstraction of a database query this is the best way that I could find to accomplish this.

Reply:
Got it. Would it be worth trying to use our own SQL query here? Don't want to add too much work (not even sure if it can be done in a query), but it would be faster. Also don't want to introduce an injection threat when this works just fine.

Reply:
Ok, @iccowan - I found a way to accomplish this without any SQL gymnastics. Here's how I did it:
Used the existing SQL query's result by making a copy of the collection and turning it into an associative array
Extracted keys from the array
Sorted the array (required so it's not possible to look at the raw HTML and correlate names to CIDs)
Pushed the CIDs back into the collection via previously written code

Thoughts? Just pushed this to the branch. It's a few extra lines of code, but eliminates the need for a second database query.